sqlglot/dialects/bigquery.py

```diff
@@ -40,6 +40,11 @@
 logger = logging.getLogger("sqlglot")
 
 
+JSON_EXTRACT_TYPE = t.Union[exp.JSONExtract, exp.JSONExtractScalar, exp.JSONExtractArray]
```

Comment:
Should we just update the `dialect.py` constant?

Reply:
This was initially like that, turns out that the `dialect.py` constant has to do with `exp.Binary` (e.g `self.binary(...)`) so adding `exp.JSONExtractArray` would lead to type errors.